packages/integrations/components/integrationsManager/componentMap.js

```diff
@@ -1,13 +1,15 @@
 // Integration components.
 import CoralEmbed from '../coral';
 import YoastSchema from '../yoastSchema';
+import SiteStats from '../siteStats';
```

Comment:
Can we call all of this `JetpackSiteStats` so it's clearer? Backend and frontend.

Reply:
👍

Reply:
I'll include that update as part of my package changes for the `withPico` coral wrapper so that I don't have to switch contexts in my current work.